src/Illuminate/Validation/Validator.php

```diff
@@ -1552,6 +1552,46 @@ protected function validateImage($attribute, $value)
     }
 
     /**
+     * Validate the dimensions of an image matches the given values.
+     *
+     * @param  string $attribute
+     * @param  mixed $value
+     * @param  array $parameters
+     * @return bool
+     */
+    protected function validateImageDimensions($attribute, $value, $parameters)
+    {
+        if (! $sizeDetails = getimagesize($value->getRealPath())) {
+            return false;
+        }
+
+        $this->requireParameterCount(1, $parameters, 'image_dimensions');
+
+        list($width, $height) = $sizeDetails;
+
+        $parameters = $this->parseNamedParameters($parameters);
+
+        if (
+            isset($parameters['width']) && $parameters['width'] != $width ||
+            isset($parameters['min_width']) && $parameters['min_width'] > $width ||
+            isset($parameters['max_width']) && $parameters['max_width'] < $width ||
+            isset($parameters['height']) && $parameters['height'] != $height ||
+            isset($parameters['min_height']) && $parameters['min_height'] > $height ||
+            isset($parameters['max_height']) && $parameters['max_height'] < $height
+        ) {
+            return false;
+        }
+
+        if (isset($parameters['ratio'])) {
+            list($numerator, $denominator) = array_pad(sscanf($parameters['ratio'], '%d/%d'), 2, 1);
+
+            return eval("return {$numerator}/{$denominator};") == $width / $height;
```

Comment:
Yes, I was using it for a different implementation but forgot to change. Will fix.

Sent from my iPhone

> On May 4, 2016, at 9:09 PM, Graham Campbell <notifications@github.com> wrote:
> 
> In src/Illuminate/Validation/Validator.php:
> 
> > +
> > +        if (
> > +            isset($parameters['width']) && $parameters['width'] != $width ||
> > +            isset($parameters['min_width']) && $parameters['min_width'] > $width ||
> > +            isset($parameters['max_width']) && $parameters['max_width'] < $width ||
> > +            isset($parameters['height']) && $parameters['height'] != $height ||
> > +            isset($parameters['min_height']) && $parameters['min_height'] > $height ||
> > +            isset($parameters['max_height']) && $parameters['max_height'] < $height
> > +        ) {
> > +            return false;
> > +        }
> > +
> > +        if (isset($parameters['ratio'])) {
> > +            list($numerator, $denominator) = array_pad(sscanf($parameters['ratio'], '%d/%d'), 2, 1);
> > +
> > +            return eval("return {$numerator}/{$denominator};") == $width / $height;
> yeh, there's no way you need an eval in there
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>